README.md

```diff
@@ -24,14 +25,16 @@ least temporarily. For example, if you are on OSX and are using Homebrew, you
 can hide METIS 5 with `brew unlink metis`. After the install procedure is
 complete, it is fine to link `metis` again with `brew link metis`.
 
-The source archive, as obtained from the HSL download process, should be placed
-as is in `Pkg.dir("HSL", "deps", "downloads")`. The `HSL` Julia module will
-take care of compilation. Once the source archives have been placed in the
-location indicated, run
+The source archive `tar.gz`, as obtained from the HSL download process, should be placed as is in `Pkg.dir("HSL", "deps", "downloads")`.
```

Comment:
We could just remove (or at least, not document) this option. Typical users will never do that. It's from the days before we registered HSL.jl.